exercises/practice/poker/tests/poker.rs

```diff
@@ -222,9 +229,37 @@ fn test_straight_flush_beats_four_of_a_kind() {
     test(&["4S 5H 5S 5D 5C", "7S 8S 9S 6S 10S"], &["7S 8S 9S 6S 10S"])
 }
 
+#[test]
+#[ignore]
+fn test_aces_can_end_a_straight_flush_high() {
+    // aces can end a straight flush (10 J Q K A)
+    test(&["KC AH AS AD AC", "10C JC QC KC AC"], &["10C JC QC KC AC"])
+}
+
+#[test]
+#[ignore]
+fn test_aces_can_end_a_straight_flush_low() {
```

Comment:
I'm not sure about "end" in this name, but the same is said of `test_aces_can_end_a_straight_low`, and I guess either end is an end

Reply:
It is called "aces can start a straight (A 2 3 4 5)" in the canonical-data.json, which is perhaps better...